en/docs/fragments/travelocity.md

```diff
@@ -15,17 +79,75 @@ Follow the instructions below to download the sample from GitHub.
 
 ### Deploy the sample web app
 
-Next, deploy the sample web app on a web container.
+Deploy this sample web app on a web container.
 
-1. Extract the `travelocity.com.war` file and open the `travelocity.properties` file located in the `<EXTRACT>/WEB-INF/classes` folder.
+1.  Copy the `travelocity.com.war`file into the `webapps` folder. For
+    example, ` <TOMCAT_HOME>/apache-tomcat-<version>/webapps`
+    
+2.  Open a terminal window and add the following entry to the
+    `           /etc/hosts          ` file of your machine to configure
+    the hostname.
 
-2.  Configure the following property with the hostname (`wso2is.local`) that you configured above. 
+    ``` bash
+    127.0.0.1   wso2is.local
+    127.0.0.1   localhost.com
+    ```
+
+    !!!info "Why is this step needed?"
+		Some browsers do not allow you to create cookies for a naked
+		hostname, such as `            localhost           `. Cookies are
```

Comment:
let's remove the unwanted places in all applicable places

Reply:
Done